sidebarComponent/client/src/components/Triangle.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import style from './Triangle.css';
+
+let Triangle = function(props) {
```

Comment:
This component seems kinda bare. Is it helpful to have it isolated as its own component?